spec/models/user_spec.rb

```diff
@@ -4,122 +4,85 @@
 
 RSpec.describe User, type: :model do
   describe 'validations' do
-    subject { user.valid? }
-
-    let(:user) { build(:user, email:, password:, name:, birth:, username:) }
+    # subject { user.valid? }
     let(:name) { 'Lucas' }
     let(:email) { 'someemail@example.com' }
     let(:password) { 'validpassword' }
     let(:birth) { Date.new(1990, 11, 0o1) }
     let(:username) { 'lucasuru' }
+    let(:random_number) { Faker::Number.between(from: 1, to: 160) }
+    let(:bio) { Faker::Lorem.paragraph_by_chars(number: random_number) }
+    let(:website) { Faker::Internet.url }
```

Comment:
You don't need this code. It's generated by faker gem on the factory. Remove them from the build(:user) unless you want to test a specific case with them